test/fixtures/cross_agent_tests/distributed_tracing/trace_context.json

```diff
@@ -1311,7 +1311,7 @@
           "tracestate.sampled",
           "tracestate.priority"
         ],
-        "vendors": [
+        "tracingVendors": [
```

Comment:
I couldn't quite tell from the PR description, but I think this change was also made in the cross-agent-tests repo. If it was, hooray! If it wasn't, please update them there before merging, otherwise this change will be overridden the next time we pull down the tests.

Reply:
An internal agent tests repo PR is now in flight. As discussed offline, merging this one now.